src/components/CommonsLogin/CommonsLogin.css

```diff
@@ -0,0 +1,16 @@
+.commons-login {
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+}
+
+.commons-login__title {
+  font-weight: 600;
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+.commons-login__logo {
+  max-height: 80px;
+  margin-bottom: 10px;
```

Comment:
This logo also seems being stretched horizontally, adding following css script will fix this: 
```
    margin-left: auto;
    margin-right: auto;
```